run_bot.py

```diff
@@ -107,4 +108,15 @@ async def main() -> None:
         group.create_task(bot_dispatcher.start_polling(bot_instance, handle_signals=False))
 
 if __name__ == "__main__":
+    parser = argparse.ArgumentParser(prog="heavens-gate", description="Run bot with core service.")
+    parser.add_argument("-awg", "--amnezia",
+                        default=False,
```

Comment:
`action='store_true'` is specifically designed for boolean flags that are false by default, use it instead of `default=False` and `action=argparse.BooleanOptionalAction`

see https://docs.python.org/3/library/argparse.html#action